app/src/main/java/com/mobilecoronatracker/ui/cumulatedreport/CumulatedReportViewModelable.kt

```diff
@@ -5,8 +5,20 @@ import androidx.lifecycle.MutableLiveData
 
 interface CumulatedReportViewModelable {
     val cases: LiveData<String>
+    val active: LiveData<String>
     val deaths: LiveData<String>
     val recovered: LiveData<String>
+    var casesCount: Int
+    var activeCount: Int
+    var deathsCount: Int
+    var recoveredCount: Int
+    val todayUpdated: LiveData<Void>
```

Comment:
Same as for history update, I would change it to LiveData<TodayDonutData> with all fields calculated and observe it instead of todayUpdate

Reply:
done